eos-updater/apply.c

```diff
@@ -255,6 +334,15 @@ apply_internal (ApplyData     *apply_data,
                local_error->message);
   g_clear_error (&local_error);
 
+  /* Try to add a collection ID to the OS remote, which enables LAN/USB OS
+   * updates when combined with eos-updater.conf changes. Don't treat any
+   * errors as fatal to the update. */
+  /* FIXME: Remove this function call after the next checxpoint */
```

Comment:
Nitpick: Probably should also include a version number here, because at any point in time we will never actually reach the ‘next checkpoint’. i.e. `FIXME: Remove this function call after the first checkpoint after 3.4.8`.